package/MDAnalysis/topology/MOL2Parser.py

```diff
@@ -93,6 +97,10 @@ def parse(self, **kwargs):
         Returns
         -------
         A MDAnalysis Topology object
+
+
+        .. versionchanged: 2.0.0
+           Parse elements from atom types.
```

Comment:
Can you add Elements to the attribute list in the docstring? Maybe with a comment about how it's read based on Sybyl atom types?